src/MadLibs.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+namespace App\Cli;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Formatter\OutputFormatterStyle;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+use Symfony\Component\Filesystem\Exception\FileNotFoundException;
+use Symfony\Component\Finder\Finder;
+
+
+//#[AsCommand(name: 'MadLibs')]
+class MadLibs extends Command
+{
+
+    protected static $defaultName = 'MadLibs';
+
+    private string $mad_libs_filename = 'madlibs.json';
+
+
+    protected function configure(): void
+    {
+        $this->setDescription("Allows user to input mad libs to the console.");
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        // find mad libs file
+        $finder = new Finder();
+        $finder->files()->in('src/data')->name($this->mad_libs_filename);
+
+        # if file not found, throws FileNotFoundException
+        if ($finder->hasResults()) {
+            // iterate through Finder object to retrieve the file
+            foreach ($finder as $file) {
+
+                if ($file->getFilename() == $this->mad_libs_filename) {
+                    $madLibsFileContents = $file->getContents();
+                    $madLibsArray = json_decode($madLibsFileContents, true);
+                    $madLibsOptions = array(); # to be: array of indexed mad libs titles
+
+                    // build madLibsOptions array of indexed titles
+                    $keyIndex = 0;
+                    foreach ($madLibsArray as $madLib) {
+                        $value = $madLib["title"];
+                        $option = array($keyIndex => $value);
+
+                        $madLibsOptions = array_merge($madLibsOptions, $option);
+                        $keyIndex++;
+                    }
+
+                    // print the titles and indices of the mad libs to the console
+                    $this->printTitleAndOptions($input, $output, $madLibsOptions);
+
+                    // PROMPT USER TO ENTER INDEX OF MAD LIBS SELECTION
+                    // CHECK IF THE SELECTED INDEX IS WITHIN THE RANGE
+                    $selectionIndex = -1;
+                    while ($selectionIndex < 0 || $selectionIndex > count($madLibsOptions)) {
+                        $selectionIndex = $this->getSelection($input, $output);
+                        if ($selectionIndex < 0 || $selectionIndex >= count($madLibsOptions)) {
+                            $output->writeln('<error>ERROR: You must enter a value between [0] and '
```

Comment:
I'm honestly not sure how PHP works with multi-line strings, if there's an easier way to do it but for string interpolation you can do:
```
$maxOptionLength = count($madLibsOptions) - 1;
$output->writeln("<error>ERROR: You must enter a value between [0] and {$maxOptionLength}. Please try again.</error>");
```